support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -92,27 +98,44 @@ export function CheckoutNudgeContainer({
 
 	let title, subtitle, paragraph;
 
-	if (dynamic) {
+	const calcWeeklyAmount = (annualAmount: number, toNearest?: number) => {
+		const weeklyAmount = Math.ceil((annualAmount * 100) / 52);
+
+		// Current/control solution is to round to the nearest 10p so this may be
+		// disposable if the variant is adopted as we want the dynamic version to be
+		// rounded to the nearest 1p
+		const roundedWeeklyAmount =
+			toNearest !== undefined
+				? Math.round(weeklyAmount / toNearest) * toNearest
+				: weeklyAmount;
+
+		return countryGroupId === 'GBPCountries' && roundedWeeklyAmount < 100
+			? roundedWeeklyAmount.toString() + `p`
+			: currencyGlyph +
+					(roundedWeeklyAmount / 100).toFixed(
+						roundedWeeklyAmount % 100 === 0 ? 0 : 2,
+					);
```

Comment:
```suggestion
		if (countryGroupId === 'GBPCountries' && roundedWeeklyAmount < 100) {
			return `${roundedWeeklyAmount.toString()}p`;
		} else {
			const fixedDigitAmount = roundedWeeklyAmount % 100 === 0 ? 0 : 2;
			const amountString = (roundedWeeklyAmount / 100).toFixed(fixedDigitAmount);
			return `${currencyGlyph}${amountString}`
		}
```
Proper if statements are always preferable to ternaries unless the condition is very simple- this is _much_ easier to read and understand for someone new to the code. Similarly extracting individual calculations to named variables makes the code a lot easier to follow.

We should also avoid ES5-style string concatenation in favour of template literals.